### PR TITLE
Add ODroid C4 support

### PR DESCRIPTION
**Pull request by xurtis**
Adds the ODroid C4 as a valid device.

### CODE REVIEW

File: elfloader-tool/src/plat/odroidc4/platform_init.c [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+#include <autoconf.h>
+#include <mode/arm_generic_timer.h>
+
+/* Reset the virtual offset for the platform timer to 0 */
+void platform_init(void)
+{
+    reset_cntvoff();
+}
+
+#if CONFIG_MAX_NUM_NODES > 1
```
Comment (axel-h) [inline elfloader-tool/src/plat/odroidc4/platform_init.c:16]: There is the alias `ENABLE_SMP_SUPPORT` that is also in use, is there a policy when to use what?

Comment (axel-h) [reply elfloader-tool/src/plat/odroidc4/platform_init.c:16]: On a 2nd look, seems this is only used in the kernel, but not in the ELF loader. So ignore that comment above.

Comment (xurtis) [reply elfloader-tool/src/plat/odroidc4/platform_init.c:16]: I think `ENABLE_SMP_SUPPORT` is only defined inside the kernel.

Comment (kent-mcleod) [reply elfloader-tool/src/plat/odroidc4/platform_init.c:16]: It used to be, but then the kernel's private config file was moved into libsel4: https://github.com/seL4/seL4/blob/master/libsel4/include/sel4/config.h#L96

Comment (axel-h) [reply elfloader-tool/src/plat/odroidc4/platform_init.c:16]: Could be a thing for later then: https://github.com/seL4/seL4_tools/issues/96

**Review by nomadeel — APPROVED** [review]

**Comment by nomadeel** [comment]
Looks good to me, the `CONFIG_MAX_NUM_NODES` could be resolved in a different PR instead.

**Pull request merged**